pkg/proxy/userspace/proxier_openshift.go

```diff
@@ -0,0 +1,13 @@
+package userspace
```

Comment:
We absorbed the userspace proxy code into sdn itself in 4.11 (https://github.com/openshift/sdn/pull/393) so the vendored `pkg/proxy/userspace` is no longer used. I think the copy in sdn should have all the modifications we need already.